### ISSUE TITLE
Openzwave 1.6 Empty groups

### ISSUE THREAD

**Issue by sincze** [domoticz/domoticz#3385]
I updated to latest domoticz beta by accident (1.6 openzwave) , had the 1.4 

Domoticz V4.10959 OpenZWave USB Version: 1.6-808-gbb0a4bf7-dirty 

Some of my ZWAVE devices are not working (updating power usage, showing their zwave config) anymore and apparantly do not belong to a group.

Unplugging and replugging the device in the socket did not help as well.

![image](https://user-images.githubusercontent.com/5776333/60757609-2f4b6500-a00d-11e9-8f78-73b2b273f95c.png)

Domoticz logs shows:

`2019-07-06 16:07:10.476 Error: OpenZWave: Value_Changed: Tried adding value, not succeeded!. Node: 10 (0x0a), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-07-06 16:07:10.476 Error: OpenZWave: Value_Changed: Tried adding value, not succeeded!. Node: 10 (0x0a), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-07-06 16:07:10.532 Error: OpenZWave: Value_Changed: Tried adding value, not succeeded!. 
`

![image](https://user-images.githubusercontent.com/5776333/60757686-fb247400-a00d-11e9-9638-766940daf273.png)

What did I miss.

**Comment by GravityRZ** [comment]
from 1.4 to 1.6 is a big jump and not without problems(have had them myself)

my best educated guess

stop domoticz
delete the ozwcache_xxxx.xml file (somewhere in domoticz/var directory)
delete the old 1.4 cache file zwcfg_xxxxxx.xml

wake up al your devices(the ones on battery)
startup domoticz

hopefully things will run better.

**Comment by sincze** [comment]
- Ok issued:

`domoticz-pi:~/domoticz/Config $ sudo /etc/init.d/domoticz.sh stop`

`domoticz-pi:~/domoticz/Config $ mv ozwcache_0xd3462876.xml _ozwcache_0xd3462876.xml.backup`
`domoticz-pi:~/domoticz/Config $ mv zwcfg_0xd3462876.xml _zwcfg_0xd3462876.xml.backup`
`domoticz-pi:~/domoticz/Config $ mv zwcfg_0xd3cf129c.xml _zwcfg_0xd3cf129c.xml.backup`

- Run around the house and slam doors.

`domoticz-pi:~/domoticz/Config $ sudo /etc/init.d/domoticz.sh start`

- Waiting.

**Comment by sincze** [comment]
Initial result after 22 minutes.

![image](https://user-images.githubusercontent.com/5776333/60760044-dd1b3b80-a02e-11e9-8f01-ed72b1bd5496.png)

**Comment by sincze** [comment]
Still a lot of HW missing or indicated with ID and tyoe 0x0000.

![image](https://user-images.githubusercontent.com/5776333/60760224-17d2a300-a032-11e9-9808-5f41870e14dd.png)

**Comment by sincze** [comment]
I did the same thing... And all start with 0x000 but now the FIB(aro) are revived... Pretty strange / vague on how to revive your complete network.

![image](https://user-images.githubusercontent.com/5776333/60761018-69822a00-a040-11e9-9816-abc27f4cca09.png)

**Comment by sincze** [comment]
Motion Sensors still unavailable same for doors, even if i move in front of them / open the door. Press the small button inside of the sensor...
![image](https://user-images.githubusercontent.com/5776333/60761029-b49c3d00-a040-11e9-9076-3ede26bebc9d.png)

**Comment by GravityRZ** [comment]
i think the devicefiles maybee do not exist.
these are the ones i have
if youre id's are not in the manufacturerspecific file or do not have config files then you need to go back to 1.4 or change the files yourself

![image](https://user-images.githubusercontent.com/16853535/60765320-1a1c1800-a099-11e9-8cbd-0b5fe00fdd15.png)

` <Manufacturer id="021A" name="SHENZHEN AOYA INDUSTRY"></Manufacturer>
  <Manufacturer id="0286" name="Shenzhen Easyhome Technology"></Manufacturer>
  <Manufacturer id="021C" name="Shenzhen iSurpass Technology"></Manufacturer>
  <Manufacturer id="037A" name="Shenzhen JBT Smart Lighting"></Manufacturer>
  <Manufacturer id="021D" name="Shenzhen Kaadas Intelligent Technology"></Manufacturer>
  <Manufacturer id="0211" name="Shenzhen Liao Wang Tong Da Technology"></Manufacturer>
  <Manufacturer id="0406" name="Shenzhen Meian Technology"></Manufacturer>
  <Manufacturer id="0258" name="Shenzhen Neo Electronics Co Ltd">
    <Product config="shenzen_neo/nas-ds01z.xml" id="0082" name="Door/Window Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ds01z.xml" id="1082" name="Door/Window Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ds01z.xml" id="2082" name="Door/Window Detector" type="0003"/>
    <Product config="shenzen_neo/nas-wr01z.xml" id="0087" name="Power plug 12A" type="0003"/>
    <Product config="shenzen_neo/nas-wr01z.xml" id="1087" name="Power plug 12A" type="0003"/>
    <Product config="shenzen_neo/nas-wr01ze.xml" id="1027" name="Power plug 12A" type="0200"/>
    <Product config="shenzen_neo/nas-wr01z.xml" id="2087" name="Power plug 12A" type="0003"/>
    <Product config="shenzen_neo/nas-pd01z.xml" id="0083" name="Battery Powered PIR Sensor" type="0003"/>
    <Product config="shenzen_neo/nas-pd01z.xml" id="1083" name="Battery Powered PIR Sensor" type="0003"/>
    <Product config="shenzen_neo/nas-pd02z.xml" id="108d" name="Battery Powered PIR Sensor V2" type="0003"/>
    
    <Product config="shenzen_neo/nas-ws02z.xml" id="0085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ws02z.xml" id="1085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ws02z.xml" id="6085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ws02z.xml" id="2085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ab01z.xml" id="0088" name="Siren Alarm" type="0003"/>
    <Product config="shenzen_neo/nas-ab01z.xml" id="1088" name="Siren Alarm" type="0003"/>
    <Product config="shenzen_neo/nas-rc01z.xml" id="008a" name="SOS/Remote Control" type="0003"/>
    <Product config="shenzen_neo/nas-rc01z.xml" id="108a" name="SOS/Remote Control" type="0003"/>
    <Product config="shenzen_neo/ls02ch.xml" id="008b" name="Wall Switch (2 channels)" type="0003"/>
    <Product config="shenzen_neo/ls02ch.xml" id="108b" name="Wall Switch (2 channels)" type="0003"/>
    <Product config="shenzen_neo/ls01ch.xml" id="008c" name="Wall Switch (1 channel)" type="0003"/>
    <Product config="shenzen_neo/ls01ch.xml" id="108c" name="Wall Switch (1 channel)" type="0003"/>
  </Manufacturer>
  <Manufacturer id="032C" name="Shenzhen Saykey Technology"></Manufacturer>
  <Manufacturer id="036C" name="Shenzhen Sen5 Technology"></Manufacturer>
  <Manufacturer id="0381" name="Shenzhen Thingsview Tech"></Manufacturer>
  <Manufacturer id="0250" name="Shenzhen Tripath Digital Audio Equipment"></Manufacturer>
  <Manufacturer id="0356" name="Shenzhen ZHIQU Technology Limited"></Manufacturer>`

**Comment by sincze** [comment]
Tx for reply, there is a file here:
`/domoticz/Config/manufacturer_specific.xml`

It has so it seems the Shenzen sensors in it.

```
Manufacturer id="0258" name="Shenzhen Neo Electronics Co Ltd">
    <Product config="shenzen_neo/nas-ds01z.xml" id="0082" name="Door/Window Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ds01z.xml" id="1082" name="Door/Window Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ds01z.xml" id="2082" name="Door/Window Detector" type="0003"/>
    <Product config="shenzen_neo/nas-wr01z.xml" id="0087" name="Power plug 12A" type="0003"/>
    <Product config="shenzen_neo/nas-wr01z.xml" id="1087" name="Power plug 12A" type="0003"/>
    <Product config="shenzen_neo/nas-wr01ze.xml" id="1027" name="Power plug 12A" type="0200"/>
    <Product config="shenzen_neo/nas-wr01z.xml" id="2087" name="Power plug 12A" type="0003"/>
    <Product config="shenzen_neo/nas-pd01z.xml" id="0083" name="Battery Powered PIR Sensor" type="0003"/>
    <Product config="shenzen_neo/nas-pd01z.xml" id="1083" name="Battery Powered PIR Sensor" type="0003"/>
    <Product config="shenzen_neo/nas-pd02z.xml" id="108d" name="Battery Powered PIR Sensor V2" type="0003"/>
    <!--<Product type="0003" id="108d" name="Battery Powered PIR Sensor with temp" config="shenzen_neo/nas-pd01z.xml"/>-->
    <Product config="shenzen_neo/nas-ws02z.xml" id="0085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ws02z.xml" id="1085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ws02z.xml" id="6085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ws02z.xml" id="2085" name="Water Leakage Detector" type="0003"/>
    <Product config="shenzen_neo/nas-ab01z.xml" id="0088" name="Siren Alarm" type="0003"/>
    <Product config="shenzen_neo/nas-ab01z.xml" id="1088" name="Siren Alarm" type="0003"/>
    <Product config="shenzen_neo/nas-rc01z.xml" id="008a" name="SOS/Remote Control" type="0003"/>
    <Product config="shenzen_neo/nas-rc01z.xml" id="108a" name="SOS/Remote Control" type="0003"/>
    <Product config="shenzen_neo/ls02ch.xml" id="008b" name="Wall Switch (2 channels)" type="0003"/>
   <Product config="shenzen_neo/ls02ch.xml" id="108b" name="Wall Switch (2 channels)" type="0003"/>
    <Product config="shenzen_neo/ls01ch.xml" id="008c" name="Wall Switch (1 channel)" type="0003"/>
    <Product config="shenzen_neo/ls01ch.xml" id="108c" name="Wall Switch (1 channel)" type="0003"/>
  </Manufacturer>
```

And the directory

`/domoticz/Config/shenzen_neo $ ls -l`

```
-rw-r--r-- 1 pi pi 1614 May 12 16:12 ls01ch.xml
-rw-r--r-- 1 pi pi 2681 May 12 16:12 ls02ch.xml
-rw-r--r-- 1 pi pi 9418 Jun  8 14:24 nas-ab01z.xml
-rw-r--r-- 1 pi pi 5399 May 28 07:34 nas-ds01z.xml
-rw-r--r-- 1 pi pi 9273 May 28 07:34 nas-pd01z.xml
-rw-r--r-- 1 pi pi 5183 May 12 16:12 nas-pd02z.xml
-rw-r--r-- 1 pi pi 2223 May 12 16:12 nas-rc01z.xml
-rw-r--r-- 1 pi pi 5821 May 12 16:12 nas-wr01ze.xml
-rw-r--r-- 1 pi pi 9603 May 28 07:34 nas-wr01z.xml
-rw-r--r-- 1 pi pi 6317 May 28 07:34 nas-ws02z.xml

```
Looks okay right ??

**Comment by GravityRZ** [comment]
i the sensors are in there with the rifght id's (the ones they normally have) then it should work.

i have read that during the creation of the ozwcache file it is best to wake up battery powered devices.

maybee you need to read that ozw cache file and see if all the devices are in there.

what also might help is go in domoticz hardware and disable the zwave controller and after that enable it again

**Comment by sincze** [comment]
i the sensors are in there with the rifght id's (the ones they normally have) then it should work.
-> did not change these since 1.4 ofcourse ;-)

i have read that during the creation of the ozwcache file it is best to wake up battery powered devices.
-> I tried that by running around first slamming doors opening windows, move in front of the motion sensors, is that the correct way??

maybee you need to read that ozw cache file and see if all the devices are in there.
-> on it thanks for the hint.

what also might help is go in domoticz hardware and disable the zwave controller and after that enable it again
-> tried that indeed ;-0

Now in the domoticz log
```
2019-07-07 10:29:11.786 Error: ZWave hardware (34) nothing received for more than 1 Minute!....
2019-07-07 10:29:12.787 Error: Restarting: ZWave
2019-07-07 10:31:26.219 Error: ZWave hardware (34) nothing received for more than 1 Minute!....
2019-07-07 10:31:27.219 Error: Restarting: ZWave
2019-07-07 10:32:32.971 Error: OpenZWave: Type_Notification, NodeID not found internally!. HomeID: 3544590454, NodeID: 255 (0xff)
```

**Comment by GravityRZ** [comment]
ok, these were the easy fixes. maybee somebody with opezwave skills can jump in to help.

i know there were problems wirth some zwave devices not accepting 1.6 version commands

check the old config files against the new config files and see if there are any changes.

make a backup of the new ones(only the ones from your devices) and replace them with the 1.4 config files.

that helped me in one case with a qubino device which had problems with a mapping command.

**Comment by sincze** [comment]
Tnx for all the help.
At least 2 doorsensors came back ;-) now, but it seems pretty strange updating domoticz including the mandatory zwave and the whole zwave network has issues... (Powerplugs not all entrys updated / devices not recognised).

**Comment by GravityRZ** [comment]
yes i know what you mean.

i went back and forth between 14 and 1.6 because my setup did not work 100% also. finally when it did work i permanently switched.

maybee you should do the same

make a backup(name it version and zwave version)
make a backup of your scripts
see if you still have a 1.4 backup

uninstall new version.

install latest working 1.4 version
restore from 1.4 backup file(only  take the 1.4 version to be save)

that is what i did

reeinstall latest

**Comment by sincze** [comment]
Can you just go back only OpenZwave??  
As stated I accidently pressed the update Domoticz.... button without the normal prior backup. 
I think going back will then be an issue due to the domoticz database being newer.....

I also noticed: **Neo Coolcam PIR Motion**
https://github.com/OpenZWave/open-zwave/issues/1866

Luckily my top floor has Zigbee... so that still works.

![image](https://user-images.githubusercontent.com/5776333/60766393-77b86080-a0a9-11e9-980a-d993ec32a0cc.png)

**Comment by gizmocuz** [comment]
@sincze , regarding the power plugs, you need to be on the latest beta version and you are not, 10962+)
There have been many many changes! All relates to your issues
You might also need to select a node, and press the new "Refresh node info" button, then quickly wake up your node by clicking on the little button. This needs to be done in 5 seconds, or you need to increase the timeout by selecting the controller (node 1) and change the timeout value here

No you can not go back only zwave... it might help if you have remaining issues to first search here in the closed issues, and the forum, or else report them here

You could go back to the stable version by going to your installation folder and issue
./updaterelease

but as said, first get yourself on the latest beta version with

./updatebeta

**Comment by petergebruers** [comment]
@GravityRZ - @sincze 

> i think the devicefiles maybee do not exist.

> I also noticed: **Neo Coolcam PIR Motion**
> [OpenZWave/open-zwave#1866](https://github.com/OpenZWave/open-zwave/issues/1866)

The screenshot shows "0000" for ID and Type and this means the node has not yet reported its type. This happens when you delete your "ozwcache*" file and the device is battery operated.
Those nodes will wake up at a regular interval so eventually OZW (and Domoticz) will get the IDs and give that device a manufacturer and a name.

If you want to speed up this process, you can click the button on the device (see manual if it is 1x click or 3x click or whatever button to push)

That issue 1866 is about setting parameters to disable the "LED blink" when it detects motion, so unrelated to your problem.

**Comment by sincze** [comment]
@gizmocuz, sorry updated to Domoticz V4.10962 this morning ;-) as I noticed something was done with  zwave (lux). Don't have that device but I thought it was good to do the update. By using the beta releases I try to help the community solving issues. But this is the biggest issue I had since 2013.

@petergebruers, thnx I indeed removed ozwcache*. As mentioned in this topic. Some Powerplugs also did not report their type yet (connected to 220v). However I did notice the list is getting smaller and smaller. 2 doorsensors an a motion sensor came back. ;-)

![image](https://user-images.githubusercontent.com/5776333/60771126-68ee9f80-a0e3-11e9-91fd-8ca983ec6080.png)

By using the Door_WC and WC Motion.... I would guess the sensor would report itself to the controller. But maybe that is a wrong presumption? 

Does 'network' heal do something usefull? Under Hardware zwave I also disabled 'Data Timeout' instead of having that on 1 minute.

**Comment by petergebruers** [comment]
> Some Powerplugs also did not report their type yet (connected to 220v).

It can take a while for OZW to go through all stages for all devices, but if this situation persists on mains powered nodes after 1/2 hour then it is not OK. The most likely causes: you have a communication issue or OZW thinks this node is dead. Try "Refresh Node Info" on one of those 0000 mains devices...

> Does 'network' heal do something usefull?

Short answer, no, it has to do with routing which indirectly may have some impact but the name "heal" is misleading.

> By using the Door_WC and WC Motion.... I would guess the sensor would report itself to the controller.

"Wake Up" is actually not the same as "trigger". Battery devices wake up at regular intervals, that's why they might suddenly get identified after hours. You can speed up this by looking at the manual of the device, it can be click or 3-click (eg FGSM-001) on the button near the battery, or it might be a specific button (eg Remotec ZRC-90)

**Comment by sincze** [comment]
@petergebruers your hint did the trick.

**NEO Coolcam Sensors door/window** 

`In domoticz Hardware -> Zwave Stick -> Select Device -> Press Refresh Node info -> Open the Door -> Press the button on the sensor 1x.`

**NEO Coolcam + Fibaro Powerlugs**
` in domoticz Hardware -> Zwave Stick -> Select Device -> Press Refresh Node info -> Switch On the light or device attached to the Powerplug, it will initiate a trigger.
`
![image](https://user-images.githubusercontent.com/5776333/60827539-b7b63b00-a1b0-11e9-94b2-e6e483036d5c.png)

Many thanks. Now let's see if everything works. 
Still 2 unknown devices... I guess some half paired devices from the past....

![image](https://user-images.githubusercontent.com/5776333/60827711-1a0f3b80-a1b1-11e9-9df0-981112a7e555.png)

**Comment by sincze** [comment]
Only thing I noticed... Is that devices are now in default CONFIGURATION again.

![image](https://user-images.githubusercontent.com/5776333/60828578-0cf34c00-a1b3-11e9-84a4-b2254074fbb0.png)

2. Send Meter Report was 30... ;-) 

![image](https://user-images.githubusercontent.com/5776333/60828607-22687600-a1b3-11e9-9b99-9d1e911b2ba8.png)

7. was definitely ON
8. was OFF

**Comment by sincze** [comment]
![image](https://user-images.githubusercontent.com/5776333/60836770-b5f77200-a1c6-11e9-8dd0-891b3ae89f15.png)
Some NEO Power Plugs don't have the W / kWh / V / A values anymore, others have only W or only kWh. So although the devices are now recognized I lost some capabilities. Any solution to bring those back? ->
`in domoticz Hardware -> Zwave Stick -> Select Device -> Press Refresh Node info` works if the node can be found ;-)

**Comment by Fishwaldo** [comment]
> Is that devices are now in default CONFIGURATION again.
@gizmocuz do you refresh the config params? (http://www.openzwave.com/dev/classOpenZWave_1_1Manager.html#aa35d05ad94fc7dcfb797de127e114080) as OZW will not get the latest when restarting (it generates too much traffic)

**Comment by gizmocuz** [comment]
@Fishwaldo , because of the latest changes in OZW 1.6 in the meter class I had to drop (1 time) the cache file as otherwise a incorrect index is read from the old cache file.

But this will NOT change any settings inside a node...

Sinsze should refresh the configuration parameters from the node (we have a button/option for this shown two posts up)

**Comment by sincze** [comment]
For all devices I need to select them and press:
![image](https://user-images.githubusercontent.com/5776333/60985094-2922f480-a33d-11e9-8f45-37d40c0f00f9.png)
Ok.

**Comment by gizmocuz** [comment]
@sincze correct.. all ok now?

**Comment by sincze** [comment]
Seems I'm back in the Domoticz game. 
Took several days and a lot of domoticz restarts... (crashed due to some zwave action, maybe pressed refresh node to often)
![image](https://user-images.githubusercontent.com/5776333/60996015-1831ad80-a354-11e9-9fde-de2f69926f10.png)

1 Node did not want to come back with the values so moved that next to the controller plugged it in... and came back to life.. Moved it back to original location..

![image](https://user-images.githubusercontent.com/5776333/60995978-02bc8380-a354-11e9-8ea9-5a22bda4bb14.png)

Now only 1 stupid dead non existing node left. It's left domoticz... (selected it an pressed delete) but OZW does not want to let it go.. yet...

**Comment by gizmocuz** [comment]
@sincze , great to hear !
If you disable the openzwave hardware, and enable it again, it is still left in domoticz, or does it appear again ?
You did try to select the node and pressed the delete button ? Or after a disable/enable delete it with the control panel ?

**Comment by sincze** [comment]
I removed it from Domoticz via -> select node -> Delete button.
It is removed from Domoticz however still visible in the control panel.

As my domoticz error log is filled with:

```
2019-07-12 21:14:19.926 Error: OpenZWave: Value_Changed: Tried adding value, not succeeded!. Node: 10 (0x0a), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-07-12 21:14:36.267 Error: OpenZWave: Value_Changed: Tried adding value, not succeeded!. Node: 25 (0x19), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-07-12 21:14:51.201 Error: OpenZWave: Value_Changed: Tried adding value, not succeeded!. Node: 10 (0x0a), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
```
It might be best to update to latest beta first to have that solved as well.. still on Domoticz V4.10962
as mentioned by https://github.com/domoticz/domoticz/issues/3360 -> ROV28, this might still be an issue that I can check.

**Comment by gizmocuz** [comment]
@sincze , you need to update to the latest beta version for that log error/message,
but also try to delete the node from the control panel
I have one zwave stick with such a problem, it looks like it is in frozen/locked state, even hard reset does not do anything

**Comment by sincze** [comment]
Did the following
![image](https://user-images.githubusercontent.com/5776333/61169014-f6ecdf00-a556-11e9-8ce1-b25f0219e780.png)
Device is reported as 

> CacheLoad (sleeping)

Next stop make backup and update to latest beta. > Domoticz V4.10988

**Comment by sincze** [comment]
After upgrading to latest beta..
![image](https://user-images.githubusercontent.com/5776333/61169558-7088cb00-a55f-11e9-89cf-73066ca7423c.png)
Here we go again, at least now we know what to do...
And the annoying sensor is back in Domoticz..
![image](https://user-images.githubusercontent.com/5776333/61169581-a75ee100-a55f-11e9-81bd-9a91b260f354.png)

**Comment by petergebruers** [comment]
> Here we go again, at least now we know what to do...

Yes, and sorry about that... Something went wrong with communication. But I've tried to fix that by posting a statement...

I've issued a warning about this update (plus suggestion to revert if issues remain). See:

#3530

Explains why you get 0x0000 , it regenerates ozwcache

> And the annoying sensor is back in Domoticz..

You say "It is removed from Domoticz however still visible in the control panel." which suggests to me that Node 14 does not really exists anymore, but your controller (dongle) has not deleted it, and it will come back each time you restart (because the controller says 14 is listed in its memory). If it is a dead (failed) node, you can try "Remove Failed Node" in ozwcp.

**Comment by sincze** [comment]
@petergebruers, missed that one,  a well this post is very usefull afterall. I learned a lot.
- Select item in the Domoticz list and press the famous 'Refresh Node Info' button.
- For all my battery powered sensors.. I walked to them opened the case press 'Refresh Node Info' and the small button inside 1x to bring them back.

Now let;s see if all properties of the devices like KwH/W/A/V are back as well l-)

**Comment by sincze** [comment]
![image](https://user-images.githubusercontent.com/5776333/61170674-15f66b80-a56d-11e9-9a90-9b8b68c74037.png)
It is a shame that all devices lost their name and location, but that was cleared by @petergebruers, nice to see power swithces now not only show on/off bute real 'current' values as well.

**Comment by sincze** [comment]
I had everything working on Domoticz V4.10988, 
- disabled the zwave stick in domoticz
- enabled the stick. (just selected the zwave hardware and pressed update).

The whole circus 0000 started again. 
Pfff. For some reason 1 Fibaro plug is not detected anymore.. mmm

ozw*****.xml
```
<?xml version="1.0" encoding="utf-8" ?>
<Driver xmlns="https://github.com/OpenZWave/open-zwave" version="4" revision="38" home_id="0xd*******" node_id="1" api_capabilities="0" controller_capabilities="8" poll_interval="60000" poll_interval_between="false">
```

**Comment by sincze** [comment]
Currently on Domoticz V4.10988 zwave 1.6-808-gbb0a4bf7-dirty
Re-interviewing the door sensors... 
Motion Sensors came back from 0000 without action.
Contact Sensors (door/window) needs a walk through the house opening the case press 'Refresh Node Info' and the small button inside 1x to bring them back.

Domoticz log only has 1 error message regading zwave ;-) that is an improvement.
`2019-07-14 02:46:06.042 Error: OpenZWave: Value_Changed: Unhandled value type ZDTYPE_SWITCH_NORMAL (2). Node: 2 (0x02), CommandClass: SENSOR MULTILEVEL, Label: Unknown, Instance: 1, Index: 0 (OpenZWave.cpp:2999)`

**0x02** is a FIBARO System FGWPE/F Wall Plug Gen5+

Still lost 1 FIBARO .... plug 0x03 it is not detected anymore by domoticz and zwave control panel.

**Comment by sincze** [comment]
To solve the Lost fibaro...

1. Unplug Fibaro plug from the power socket remove monitored device from powerplug
2. Plug only the plug back in, ring will glow green to indicate it is already paired (see manual) to zwave
3. In Domoticz -> Zwave -> Include Node (secure).
4. Tripple press the button on the power plug... and wait.

The Fibaro plug was added as device 0x03.

**Comment by sincze** [comment]
Seems stable now PI with Domoticz V4.10988
OpenZWave USB Version: 1.6-808-gbb0a4bf7-dirty 
`<Driver xmlns="https://github.com/OpenZWave/open-zwave" version="4" revision="38" home_id="0xd*******" node_id="1" api_capabilities="0" controller_capabilities="8" poll_interval="60000" poll_interval_between="false">`

![image](https://user-images.githubusercontent.com/5776333/61319401-77396b80-a807-11e9-96e3-457f58b03820.png)

Still having 1 node that does not want to leave or mark itself DEAD.... but you can't have it all.

Domoticz error.log incidental errors only.
```
2019-07-14 02:46:06.042 Error: OpenZWave: Value_Changed: Unhandled value type ZDTYPE_SWITCH_NORMAL (2). Node: 2 (0x02), CommandClass: SENSOR MULTILEVEL, Label: Unknown, Instance: 1, Index: 0 (OpenZWave.cpp:2999)
2019-07-16 03:31:07.942 Error: OpenZWave: Value_Changed: Unhandled value type ZDTYPE_SWITCH_NORMAL (2). Node: 5 (0x05), CommandClass: SENSOR MULTILEVEL, Label: Distance, Instance: 1, Index: 20 (OpenZWave.cpp:2999)
```
Node 0x02 and 0x05 are FIBARO wall switches power plugs (colour rings)

**Comment by gizmocuz** [comment]
@sincze , great it is working. For the Dead node that can not be removed (also not using the control panel),  maybe it's best to open a another issue (or on the openzwave issue page)

Regarding those two log lines... that is strange indeed...
0x02 = decimal

Label Unknown... Label Distance ? Those do not belong there.... maybe refreshing the node information would help ?

Closing this issue, as this is solved

**Comment by sincze** [comment]
Tnx for all the support this ticket shows indeed what you can do to bring the ZWAVE network back to life.
![image](https://user-images.githubusercontent.com/5776333/61646575-b3a71480-acaa-11e9-88f4-c048dcf7bed0.png)

**Comment by gizmocuz** [comment]
@sincze , thanks for the feedback, but what exactly did you do ? ;)

**Comment by sincze** [comment]
Just a wrap up of this incident.

I updated to latest Domoticz beta by accident (1.6 openzwave) , had the 1.4
All devices had 0000 for ID, this means the node has not yet reported its type. 
This happens when you delete your "ozwcache*" file and the device is battery operated.
Those nodes will wake up at a regular interval so eventually OZW (and Domoticz) will get the IDs and give that device a manufacturer and a name.

To fix it:

**NEO Coolcam Sensors door/window (Battery Nodes)**

In Domoticz Hardware -> Zwave Stick -> Select Device -> Press Refresh Node info -> Press the button on the sensor 1x.

**NEO Coolcam + Fibaro Powerlugs (Powered Nodes)**
in Domoticz Hardware -> Zwave Stick -> Select Device -> Press Refresh Node info -> Switch On the light or device attached to the Powerplug, it will initiate a trigger.

Now the configuration in Domoticz and the plugs are not in sequence, to fix this..
Select the device and press 
![image](https://user-images.githubusercontent.com/5776333/61699027-6ffbeb80-ad3a-11e9-9b77-d25069f2461b.png)

After a while all data is in Sync again.

I lost 1 node permanently that could be brought back to life by re-including it near the Zwave stick. 
btw I use a secure (node) network, I found a hint suggesting that new nodes should always be included NEAR the zwave stick and not as described in the wiki on the final install location... as then the sensor could be out of range to do the initial **secure** pairing.
